### PR TITLE
add definition of toast, using quotes, etc.

### PR DESCRIPTION
**Pull request by monico-moreno**
Given questions like in issue #26, added a definition to the readme to address these questions from the community.

### CODE REVIEW

**Comment by domenic** [comment]
This is great, thank you! Do you think maybe the last paragraph should reference the libraries researched in https://github.com/jackbsteinberg/std-toast/blob/master/study-group/ ?

**Comment by monico-moreno** [comment]
You're welcome!  😊

Changes made to reference the study group page. 👍

**Comment by domenic** [comment]
@edent, does this help? In my opinion, we should merge this to close #26 and make it clearer what the repository is about, but open a new issue to discuss whether "toast" is the _best_ name. How does that sound to you?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+
+Furthermore, Ben Brocka has a nice, concise definition, in his [ux.stackexchange.com response](https://ux.stackexchange.com/a/12000), _"A Toast is a non modal, unobtrusive window element used to display brief, auto-expiring windows of information to a user."_, which adds the distinguishing detail of a toast's being **auto-expiring**.
+
+In the absence of browser-intrinsic toasts, the current state of affairs is that libraries like Bootstrap include [toast features like this one](https://getbootstrap.com/docs/4.2/components/toasts/).  Additionally, here is the Toast Study Group's [list of 3rd party libraries](https://github.com/jackbsteinberg/std-toast/tree/master/study-group) that includes an awesome table of comparison.
```
Comment (mkruisselbrink) [inline README.md]: calling your own study "awesome" seems a bit pretentious, and it also doesn't seem like including that words adds anything to the sentence? (it's also not clear to me who or what the "Toast Study Group" is)

Comment (domenic) [reply README.md]: Thanks for pointing this out. Notice that this was written by a third-party contributor---thanks @monico-moreno for the compliment! But yeah, I'll touch it up before merging.

Comment (monico-moreno) [reply README.md]: You're welcome, @domenic.   Good catch, @mkruisselbrink.  I will, however, leave the question as to what the Toast Study Group is to the experts.  Perhaps a link to a description segment or page? 🤔

**Comment by edent** [comment]
As you have screenshots, it would be great to include them on the readme. I realise the plural of anecdote is not data, but I had no idea what this was talking about until I saw images.

I'd suggest testing this readme with a few people unfamiliar with the nomenclature to see how else it can be approved.

**Comment by domenic** [comment]
Yeah, we've definitely heard the message loud and clear from the community on this. We'd so far only spoken to UI engineers or people who work on design systems, who were quite familiar; with the broader audience, we need to make things clearer.

**Comment by monico-moreno** [comment]
@domenic , @edent  I added an animated gif that shows what a typical look-and-feel would be like.

**Comment by domenic** [comment]
Thank you so much @monico-moreno! I made some minor tweaks and will now merge this, then file a follow-up issue for folks to discuss naming.

**Pull request merged**